### PR TITLE
Fixes away mission pamphlet desc

### LINKED ISSUE #13272: Gateway doesn't require direct powernet power despite being hooked up to a node.
According to the pamphlet, and how it is mapped, the gateway should require power from the grid directly and apc power. However, it only seems to use the power from the apc.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes #13272

### CODE REVIEW

**Comment by Thunder12345** [comment]
Another wasted get

**Comment by bgobandit** [comment]
>a simple desc fix
>fails travis

**Comment by Jordie0608** [comment]
Another get flawlessly wasted

File: code/modules/awaymissions/pamphlet.dm [hunk 1]
```diff
@@ -13,9 +13,8 @@
 			Nanotrasen's operational area and start your new life.<br><br>\
 			<b>Gateway Operation Basics</b><br>\
 			All Nanotrasen approved Gateways operate on the same basic principals. They operate off \
-			area equipment power as you would expect, but they also require a backup wire with at least \
-			128, 000 Watts of power running through it. Without this supply, it cannot safely function \
-			and will reject all attempts at operation.<br><br>\
+			area equipment power as you would expect,and without this supply, it cannot safely function, \
```
Comment (tkdrg) [inline code/modules/awaymissions/pamphlet.dm]: Needs a space after the first comma.

**Comment by KorPhaeron** [comment]
@tkdrg 

Fixed

**Comment by JJRcop** [comment]
Kor tried to get "buffs cakehat" to be this, all three PRs were created in the same minute!

**Pull request merged**